capstone/src/main/webapp/scripts/poll.js

```diff
@@ -0,0 +1,137 @@
+const BAR_WIDTH = "690";
+const BAR_HEIGHT = "55";
+
+function getPollOptions() {
+  fetch("/poll")
+    .then(response => response.json())
+    .then(options => {
+      options.forEach(option => {
+        createOptionElementLoad(option);
+      });
+    });
+}
+
+function createOptionElementLoad(option) {
+  let innerBar = "200";
+  let numVotes = option["userVotes"].length;
+  let text = option["text"];
+
+  const optionsContainer = document.getElementById("options-container");
+  const optionContainer = document.createElement("option-container");
+  optionContainer.setAttribute("class", "option-container");
+  //Add challenge name
+  const challengeName = document.createElement("p");
+  const challengeText = document.createTextNode(text);
+  challengeName.appendChild(challengeText);
+  optionContainer.appendChild(challengeName);
```

Comment:
Even if you do end up getting rid of the duplicate code, I think it might make sense to break out some of these bits into their own functions for readability, so that whichever one of these functions you choose to keep isn't too long. (And if you decide that you need both of the createOptionElement functions, you can still avoid duplicate code by breaking up parts into their own sections.)